### ISSUE TITLE
DataTable inside PanelGrid renders extra TD-Elements

### ISSUE THREAD

**Issue by quidam** [primefaces/primefaces#2206]
## 1) Environment
- PrimeFaces version: 6.1 RC2
- Does it work on the newest released PrimeFaces version? Works on 6.0.
- Does it work on the newest sources in GitHub? no.
- Application server + version: Wildfly 9.0.2 and 10.0
- Affected browsers: Firefox 52, Chrome 57, IE 11

## 2) Expected behavior

For each column of a DataTable one TD-Element is rendered.

## 3) Actual behavior

For each column of a DataTable two TD-Elements are rendered, one without content, one with the column's content (see screen shot below).
![extra-td](https://cloud.githubusercontent.com/assets/339391/24356306/ba99a1ce-12f9-11e7-9364-bbeac8dabbb6.png)

## 4) Steps to reproduce

Place the sample XHTML on an application server with PrimeFaces 6.1RC2 or newer, open page in browser.

## 5) Sample XHTML
```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
	xmlns:h="http://java.sun.com/jsf/html"
	xmlns:p="http://primefaces.org/ui">
	<body>
		<h:form>
			<p:panelGrid>
				<p:row>
					<p:column>
						<p:dataTable var="ent" value="#{['Test1']}">
							<p:column style="width:100px;border:1px solid;" headerText="Table Column">
								<h:outputText value="#{ent}" />
							</p:column>
						</p:dataTable>
					</p:column>
				</p:row>
			</p:panelGrid>
		</h:form>
	</body>
</html>
```
## 6) Sample bean

not required.

**Comment by tandraschko** [comment]
A PR would be great.

**Comment by quidam** [comment]
I'm not experienced in UI development, so I've got problems finding a fix that doesn't break something else. But I've found the change that causes the error:
In line 1220 of org/primefaces/component/datatable/DataTableRenderer.java
```java
column.renderChildren(context);
```
has been replaced by
```java
column.encodeAll(context);
```
which not only renders the children, but also the column (i.e. TD element), again.

This change has been introduced by you with the comment "reapplied #2138". Maybe this helps.
